packages/components/src/Form/Inputs/Checkbox/Checkbox.tsx

```diff
@@ -94,14 +120,11 @@ const CheckboxLayout = forwardRef(
         </FauxCheckbox>
       </div>
     )
-  }
-)
-
-CheckboxLayout.displayName = 'CheckboxLayout'
-
-export const Checkbox = styled(CheckboxLayout)`
+  })
+)<CheckboxProps>`
   ${reset}
   ${space}
+  ${rippleStyle}
   height: 1rem;
   position: relative;
   width: 1rem;
```

Comment:
I think if you update this to the following it should work. That makes it look really big to me, but that's what's in the spec as "Density 0": https://www.figma.com/file/ADmrXwJUbKG0eVAhco8X10/LGM2-Transition?node-id=1470%3A561
```suggestion
  height: ${({ theme: { space } }) => space.u12};
  padding: ${({ theme: { space } }) => space.u3};
  position: relative;
  width: ${({ theme: { space } }) => space.u12};
```